app/controllers/answers_controller.rb

```diff
@@ -5,11 +5,13 @@ def new
   end
 
   def create
+    params[:answer][:question_id] = params[:question_id]
```

Comment:
This is kinda weird...is there a way to make this less ... weird?  I feel like you're avoiding using ActiveRecord properly.